static/js/ducks/sources.js

```diff
@@ -93,4 +95,13 @@ const reducer = (state = initialState, action) => {
   }
 };
 
+// Websocket message handler
+messageHandler.add((actionType, payload, dispatch, getState) => {
+  const { group } = getState();
+
+  if (actionType === FETCH_GROUP_SOURCES) {
```

Comment:
I don't see this WS message type ever being pushed on the backend -- why is this needed?

Reply:
I used it when I wanted the group sources page to update automatically. 

I guess we have decided to make it non-reactive, so I can remove this handler...